src/main/java/server/router/routes/SimpleGet.java

```diff
@@ -1,27 +1,33 @@
 package server.router.routes;
 
-import java.util.List;
+import java.util.Arrays;
+import java.util.LinkedHashSet;
+import java.util.Set;
 import server.constants.Method;
-import server.constants.Path;
+import server.constants.Status;
 import server.request.Request;
+import server.response.Response;
+import server.response.ResponseBuilder;
 
-public class SimpleGet extends Route {
-  public SimpleGet(Request request) {
-    super(request);
-  }
+public class SimpleGet implements RouteHandler {
+  Set<Method> methodsAllowed = new LinkedHashSet<>(Arrays.asList(Method.GET, Method.HEAD));
 
   @Override
-  public Path path() {
-    return Path.SIMPLE_GET;
-  }
+  public Response processRequest(Request request) {
+    ResponseBuilder responseBuilder = new ResponseBuilder();
+    Method method = request.method();
 
-  @Override
-  public List<Method> methods() {
-    return List.of(Method.GET);
-  }
+    switch (method) {
+      case GET:
```

Comment:
Because [HEAD Method](https://httpwg.org/specs/rfc7231.html#HEAD) is identical to GET except body, you can write this like this:
```
switch (method) {
    case GET:
    case HEAD:
      break;
    default:
      responseBuilder.setStartLine(Status.NOT_ALLOWED.format());
      responseBuilder.addAllowHeader(methodsAllowed);
}
```